changelog.d/5984.bugfix

```diff
@@ -0,0 +1 @@
+Link send loop opentracing to edu contexts regardles of destination.
```

Comment:
Bug fix changelogs should really be in the form of `Fix ...`

```suggestion
Fix sending of EDUs when opentracing is enabled with an empty whitelist.
```

Reply:
Is there documentation for the format of a changelog?

Reply:
Not particularly, and its not a hard and fast rule, its more about having the changelog describe whats being fixed (or added, etc)